pico/repositories.bzl

```diff
@@ -39,10 +38,5 @@ def rules_pico_dependencies():
         sha256 = "329f125f681b9adbfc89fff716026a6c3dbe517f172451dd04ceb91b10e7c5e1",
     )
 
-    maybe(
-        gcc_arm_embedded_toolchain,
-        name = "gcc-arm-embedded",
-    )
-
 def rules_pico_toolchains():
-    native.register_toolchains("//toolchain:gcc-pico")
+    native.register_toolchains("//toolchain:gcc-pico_linux_x86_64")
```

Comment:
```
bazel query "@rules_pico//toolchain:*"

@rules_pico//toolchain:BUILD
@rules_pico//toolchain:cc_toolchain_darwin_x86_64_gcc-pico
@rules_pico//toolchain:cc_toolchain_linux_aarch64_gcc-pico
@rules_pico//toolchain:cc_toolchain_linux_x86_64_gcc-pico
@rules_pico//toolchain:cc_toolchain_windows_x86_64_gcc-pico
@rules_pico//toolchain:config_darwin_x86_64_gcc-pico
@rules_pico//toolchain:config_linux_aarch64_gcc-pico
@rules_pico//toolchain:config_linux_x86_64_gcc-pico
@rules_pico//toolchain:config_windows_x86_64_gcc-pico
@rules_pico//toolchain:empty
@rules_pico//toolchain:gcc-pico_darwin_x86_64
@rules_pico//toolchain:gcc-pico_linux_aarch64
@rules_pico//toolchain:gcc-pico_linux_x86_64
@rules_pico//toolchain:gcc-pico_windows_x86_64
```

Lots of generated targets. I just picked the one I wanted; I imagine something fancier is needed here.